activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryTasksIT.java

```diff
@@ -1463,4 +1464,70 @@ public void shouldGetTaskUserCandidatesAfterTaskCompleted() {
 
     }
 
+    @Test
+    public void shouldFilterTasksForDueDate() {
+        //given
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
+        sdf.setTimeZone(TimeZone.getTimeZone("GMT"));
+
+        Date dueDate = new Date();
+        Calendar cal = Calendar.getInstance();
+        cal.set(Calendar.SECOND, 0);
+        cal.set(Calendar.MILLISECOND, 0);
+        Date now = cal.getTime();
+
+        //set due date as current date + 1
+        dueDate.setTime(now.getTime() + 86400000);
```

Comment:
Maybe you could use `Duration.ofDays(1).toMillis()` instead `86400000` as we can figure out that the duration is one day without any calculation.